doc/development/language-server/debugging.rst

```diff
@@ -1,62 +1,5 @@
-Development
-===========
-
-.. contents::
-   :depth: 2
-   :backlinks: none
-   :local:
-
-Package Structure
------------------
-
-Phpactor is divided into _packages_. Each package occupies a directory in
-`lib/`. In addition there are extensions which integrate packages with
-Phpactor which are in `lib/Extension`.
-
-Tests and benchmarks are maintained in the package's directory, e.g.
-`lib/WorseReflection/Tests`.
-
-Logging
--------
-
-Logging is disabled by default, but can provide some useful information
-(such as errors encountered when parsing files etc).
-
-Enable it as follows:
-
-::
-
-   {
-       "logging.enabled": true,
-       "logging.level": "debug",
-       "logging.path": "phpactor.log",
-   }
-
-Var Dump Server
----------------
-
-Phpactor includes the Symfony Var Dumper, this allows you to inspect values
-while the server is running or an RPC request is being executed.
-
-Start the server in the Phpactor project root:
-
-::
-
-   $ ./vendor/bin/var-dump
-   ./vendor/bin/var-dump-server
-
-   Symfony Var Dumper Server
-   =========================
-
-    [OK] Server listening on tcp://127.0.0.1:9912
-
-    // Quit the server with CONTROL-C
-
-You can use the `dump` function in the code and the variable will be shown in
-the console output of the server you started: `dump($var)`.
-
 Debugging RPC
--------------
+=============
```

Comment:
this is h1? this should stay as h2 `----` if there is more than one section in the document, only the first header (h1) should use `===`